### PR TITLE
(PLATFORM-3049,PLATFORM-3050) Fix embedding Google spreadsheets

### PR DESCRIPTION
**Pull request by Grunny**
Fix the embed URL for Google spreadsheets and cleanup arguments.

/cc @Wikia/core-platform-team

### CODE REVIEW

File: extensions/wikia/GoogleDocs/GoogleDocs.php [hunk 1]
```diff
@@ -11,41 +11,44 @@
...
 //i18n
 $wgExtensionMessagesFiles['GoogleDocs4MW'] = __DIR__ . '/GoogleDocs.i18n.php';
 
 function wfGoogleDocs( Parser $parser ) {
-	$parser->setHook( "googlespreadsheet", "renderGoogleSpreadsheet" );
+	$parser->setHook( 'googlespreadsheet', 'renderGoogleSpreadsheet' );
 	return true;
 }
 
 function renderGoogleSpreadsheet( $input, $argv ) {
```
Comment (WikiaTech) [inline extensions/wikia/GoogleDocs/GoogleDocs.php]: MAJOR Remove the unused function parameter "$argv". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: extensions/wikia/GoogleDocs/GoogleDocs.php [hunk 2]
```diff
@@ -11,41 +11,44 @@
  *
  */
 
-if (!defined('MEDIAWIKI')){
-    echo ('THIS IS NOT VALID ENTRY POINT.'); exit (1);
+if ( !defined( 'MEDIAWIKI' ) ) {
+    echo( 'THIS IS NOT VALID ENTRY POINT.' );
```
Comment (WikiaTech) [inline extensions/wikia/GoogleDocs/GoogleDocs.php]: MAJOR Remove the parentheses from this "echo" call. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/GoogleDocs/GoogleDocs.php [hunk 3]
```diff
@@ -11,41 +11,44 @@
...
 //i18n
 $wgExtensionMessagesFiles['GoogleDocs4MW'] = __DIR__ . '/GoogleDocs.i18n.php';
 
 function wfGoogleDocs( Parser $parser ) {
-	$parser->setHook( "googlespreadsheet", "renderGoogleSpreadsheet" );
+	$parser->setHook( 'googlespreadsheet', 'renderGoogleSpreadsheet' );
 	return true;
 }
 
 function renderGoogleSpreadsheet( $input, $argv ) {
```
Comment (WikiaTech) [inline extensions/wikia/GoogleDocs/GoogleDocs.php]: MAJOR Remove the unused function parameter "$argv". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by harnash — APPROVED** [review]

File: extensions/wikia/GoogleDocs/GoogleDocs.php (hunk 3 above)
Comment (WikiaTech) [inline extensions/wikia/GoogleDocs/GoogleDocs.php]: MAJOR Remove the unused function parameter "$argv". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

**Review by sqreek — APPROVED** [review]

**Pull request merged**